src/01_clock_resolution.jl

```diff
@@ -21,8 +21,11 @@ function estimate_clock_resolution(n_samples::Integer = 10_000)
     for _ in 1:n_samples
         t1 = Base.time_ns()
         t2 = Base.time_ns()
-        Δt = t2 - t1
-        min_Δt = min(min_Δt, Δt)
+        # On Linux AArch64 it seems that t1 and t2 could be the same sometimes
+        if t2 > t1
+            Δt = t2 - t1
+            min_Δt = min(min_Δt, Δt)
+        end
     end
 
     min_Δt
```

Comment:
Done, with a shorter version. I think a integer comparison and a branch should be cheap enough that we can just retry a second time. It's very unlikely that this version will dead loop since that means the system clock never advances....